### PR TITLE
Make nanoemoji able to handle multiple config files

### PR DESCRIPTION
**Pull request by rsheeter**
Meant to replace https://github.com/googlefonts/color-fonts/pull/75

### CODE REVIEW

File: src/nanoemoji/nanoemoji.py [hunk 1]
```diff
@@ -83,94 +83,148 @@ def _bool_flag(name: str, value: bool):
     return flag
 
 
-def _ufo_rule(master: MasterConfig) -> str:
-    return "write_" + master.style_name.lower() + "_ufo"
+def _per_config_file(font_config: FontConfig, suffix: str) -> Path:
+    return build_dir() / Path(font_config.output_file).with_suffix(suffix).name
```
[the remark this answers is not in the document]
Comment (anthrotype) [reply src/nanoemoji/nanoemoji.py:87]: Oh I see you did that in load_configs, never mind thanks

**Review by anthrotype — APPROVED** [review]
thanks, this is awesome! ninja all the things 🥷

**Comment by anthrotype** [comment]
the test failure is unrelated to this PR, in fact I re-run the ci jobs from main branch and they also failed with the same error.

https://github.com/googlefonts/nanoemoji/runs/3892438198?check_suite_focus=true

It seems to be related to some recent picosvg changes. If I pip install picosvg==0.18.3 the test pass. I'll investigate

**Comment by anthrotype** [comment]
the culprit seems to be https://github.com/googlefonts/picosvg/pull/245, since the nanoemoji tests start to fail from picosvg 0.18.4 onwards, and the only diff since 0.18.3 is that PR

**Comment by anthrotype** [comment]
i'm not sure what actually changed but I think the diff is benign so I'm inclined to simply accept it and adjust the expected result

```xml
-          <dy value="-28.85938"/>
+          <dy value="-28.859"/>
```

**Comment by anthrotype** [comment]
hm there's another test failure in tests/glyph_reuse_test.py which seems more worrying: it fails to find a reuse where it would find it before

https://github.com/googlefonts/nanoemoji/runs/3892438198?check_suite_focus=true#step:5:60

**Comment by anthrotype** [comment]
BTW, real build time after this PR on my local glinux workstation went down from 199 minutes to _just_ 47 minutes to build all the 49 test fonts from googlefonts/color-fonts repo. 🚀

**Comment by rsheeter** [comment]
Fixes #351

**Pull request merged**